container_pipeline/lib/shell/registry_delete_revision_tags.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run as root"
+   exit 1
+fi
+
+REG_PATH="/var/lib/registry/docker/registry/v2/repositories/";
+MANIFEST_LIST="/tmp/registry_manifests.txt"
+
+pushd ${REG_PATH}
+
+echo "Searching for revision tags to remove ... ";
+
+MANIFESTS_WITHOUT_TAGS=$(comm -23 <(find . -type f -name "link" | grep "_manifests/revisions/sha256" | grep -v "\/signatures\/sha256\/" | awk -F/ '{print $(NF-1)}' | sort) <(for f in $(find . -type f -name "link" | grep "_manifests/tags/.*/current/link"); do cat ${f} | sed 's/^sha256://g'; echo; done | sort))
```

Comment:
should you simplify this operation? or decorate a complex operation with comments about whats being done here?